src/hotspot/share/cds/archiveHeapLoader.hpp

```diff
@@ -66,7 +66,7 @@ class ArchiveHeapLoader : AllStatic {
   }
 
   static bool are_archived_strings_available() {
```

Comment:
The functions don't seem to carry their weight any more. Perhaps we can just rename `is_fully_available` to `is_archive_available` and always call it directly?

Reply:
We use to allow the archive to be partially loaded. This is no longer true anymore. I renamed the three functions `are_archived_strings_available`, `are_archived_mirrors_available` and `is_fully_available` to `is_in_use()`. The name corresponds to the `ArchiveHeapLoader::can_use()` function.

(It's a pity that the archive can be "used" either by "mapping" or "loading" -- see comments at the top of archiveHeapLoader.hpp. Eventually we will support only mapping. At that time,  `is_in_use()` will be renamed to `is_mapped()` ...)